test/sanity/code-smell/mypy.requirements.in

```diff
@@ -4,6 +4,7 @@ jinja2  # type stubs not published separately
 packaging  # type stubs not published separately
 pytest  # type stubs not published separately
 pytest-mock  # type stubs not published separately
+resolvelib  # this should've been included from the packaging meta
```

Comment:
Can you elaborate on what you mean by this comment?

Reply:
@mattclay normally, it's a good idea to `pip install -e .` the project under test where MyPy runs which would pick up all its runtime dependencies instead of having effort duplication.